newsfragments/2727.changed.md

```diff
@@ -0,0 +1 @@
+`PyCodeObject` is now defined with fields on Python 3.7.
```

Comment:
This needs to be `2726` not `2727`. Maybe also better to class as "fixed" instead of "changed", if this was previously working for you?

Reply:
2726 or the reported issue, 2725?

Reply:
Ah, the former.